misc/docerina/src/main/java/org/ballerinalang/docgen/generator/model/Variable.java

```diff
@@ -24,8 +24,9 @@ public class Variable extends Construct {
     @Expose
     public Type type;
 
-    public Variable(String name, String description, boolean isDeprecated, Type type) {
-        super(name, description, isDeprecated);
+    public Variable(String name, String description, boolean isDeprecated,
+                    Type type) {
+        super(name, description, null, isDeprecated);
```

Comment:
Can't we overload super constructor instead of passing `null`?

Reply:
Sure we can. I will update the PR.

Reply:
addressed review suggestions